src/backend/api/Fusion.Resources.Api/Notifications/RequestsNotificationHandler.cs

```diff
@@ -0,0 +1,263 @@
+﻿using Fusion.ApiClients.Org;
+using Fusion.Integration.Notification;
+using Fusion.Integration.Org;
+using Fusion.Resources.Api.Notifications.Markdown;
+using Fusion.Resources.Database.Entities;
+using Fusion.Resources.Domain;
+using Fusion.Resources.Domain.Notifications;
+using Fusion.Resources.Domain.Notifications.Request;
+using Fusion.Resources.Domain.Queries;
+using MediatR;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Fusion.Resources.Api.Notifications
+{
+    public class RequestsNotificationHandler :
+        INotificationHandler<RequestCreated>,
+        INotificationHandler<RequestApprovedByCompany>,
+        INotificationHandler<RequestApprovedByContractor>,
+        INotificationHandler<RequestDeclinedByCompany>,
+        INotificationHandler<RequestDeclinedByContractor>
+    {
+        private readonly IMediator mediator;
+        private readonly IFusionNotificationClient notificationClient;
+        private readonly IProjectOrgResolver orgResolver;
+        private readonly IUrlResolver urlResolver;
+
+        public RequestsNotificationHandler(IMediator mediator, IFusionNotificationClient notificationClient, IProjectOrgResolver orgResolver, IUrlResolver urlResolver)
+        {
+            this.mediator = mediator;
+            this.notificationClient = notificationClient;
+            this.orgResolver = orgResolver;
+            this.urlResolver = urlResolver;
+        }
+
+        public async Task Handle(RequestCreated notification, CancellationToken cancellationToken)
+        {
+            var request = await GetRequestAsync(notification.RequestId);
+
+            if (request == null)
```

Comment:
Expression is always false according to my visual studio...

Reply:
According to ReSharper I guess. Anyway, good catch! 
I added null handling with exception in the query instead and removed all the null checks.